src/dev-dashboard/server/static/boards-templates.md

```
@@ -0,0 +1,136 @@
+  "board": "<slug>",
+  "cards": [
+    {"ref":"s","kind":"section","payload":{"title":"Checkout"},
+     "children":["st1","st2","cl","co"]},
+    {"ref":"st1","kind":"step","payload":{"n":1,"title":"Open cart","status":"pass","cardId":<shotId>}},
+    {"ref":"st2","kind":"step","payload":{"n":2,"title":"Pay","status":"fail","cardId":<shotId>,"note":"spinner never resolves"}},
+    {"ref":"cl","kind":"checklist","payload":{"title":"Sweep","items":[
+      {"text":"empty cart state"},{"text":"invalid card","done":true},{"text":"3DS redirect"}]}},
+    {"ref":"co","kind":"callout","payload":{"tone":"warn","md":"Pay button stays enabled while the request is in flight"}}
+  ]
+}
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)